dev-tools-java/src/main/java/org/jboss/forge/addon/devtools/java/JavaFormatSourcesCommand.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2014 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.jboss.forge.addon.devtools.java;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import org.jboss.forge.addon.parser.xml.resources.XMLResource;
+import org.jboss.forge.addon.resource.FileResource;
+import org.jboss.forge.addon.resource.Resource;
+import org.jboss.forge.addon.ui.command.AbstractUICommand;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.context.UIExecutionContext;
+import org.jboss.forge.addon.ui.input.InputComponentFactory;
+import org.jboss.forge.addon.ui.input.UIInput;
+import org.jboss.forge.addon.ui.input.UIInputMany;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.result.Results;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+import org.jboss.forge.furnace.Furnace;
+import org.jboss.forge.furnace.container.simple.lifecycle.SimpleContainer;
+import org.jboss.forge.roaster.Roaster;
+
+public class JavaFormatSourcesCommand extends AbstractUICommand
+{
+
+   private final InputComponentFactory inputFactory;
+
+   private UIInput<XMLResource> profile;
+
+   @SuppressWarnings("rawtypes")
```

Comment:
@gastaldi I have previously tried to remove this annotation.
But when taking creating input via the InputComponentFactory wildcarding FileResource is not excepted.

sources = inputFactory.createInputMany("sources", 's', FileResource.class);

If removing the annotation is still possible , could you give me a pointer so that I can implement it.
All the rest of the above points I've implemented . I'll soon update my PR.